### PR TITLE
Memory

### LINKED ISSUE #168: Change default zstd-compress-level to 19
### Checklist

- [X] I added a descriptive title
- [X] I searched open requests and couldn't find a duplicate

### What is the idea?

#167 shows that our current default compression level of -22 (from the command line, guarded behind `zstd --ultra`) uses 600+ MB. -19, the highest non-ultra level, uses a little less than 100MB and will be faster to compress. Maybe the default level should be 19, or even a lower amount.

### Why is this needed?

_No response_

### What should happen?

_No response_

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by dholth**
### Description

Fixes #168

### Checklist - did you ...

- [ ] Add a file to the `news` directory ([using the template](../blob/main/news/TEMPLATE)) for the next release's release notes?
     
- [ ] Add / update necessary tests?
- [ ] Add / update outdated documentation?

### CODE REVIEW

**Comment by dholth** [comment]
This would be even better if we checked the memory usage in pytest.

**Review by mbargull — DISMISSED** [review]
Tried the same and it worked locally :).

File: src/conda_package_handling/conda_fmt.py [hunk 1]
```diff
@@ -19,10 +19,10 @@
 from .streaming import _extract
 
 CONDA_PACKAGE_FORMAT_VERSION = 2
-DEFAULT_COMPRESSION_TUPLE = (".tar.zst", "zstd", "zstd:compression-level=22")
+DEFAULT_COMPRESSION_TUPLE = (".tar.zst", "zstd", "zstd:compression-level=19")
```
Comment (dholth) [inline src/conda_package_handling/conda_fmt.py:22]: this is a backwards compatibility with libarchive tuple

**Review by jezdez — APPROVED** [review]

**Comment by jakirkham** [comment]
Am proposing setting the same default value in conda-build ( https://github.com/conda/conda-build/pull/4663 )

**Comment by dholth** [comment]
Would have liked to have fewer repeated 19's but it's awkward importing the API from the CLI and vice versa.

**Comment by jakirkham** [comment]
Yeah squashed one duplication in that PR (CLI message)

More generally though maybe we can refactor out some of this config stuff into a separate package or perhaps into Conda itself for broader reuse

**Pull request merged**